iotdb/src/main/java/org/apache/iotdb/db/conf/directories/strategy/MaxDiskUsableSpaceFirstStrategy.java

```diff
@@ -26,7 +26,7 @@
 public class MaxDiskUsableSpaceFirstStrategy extends DirectoryStrategy {
 
   // disk space is measured by MB
-  private final long dataSizeShift = 1024 * 1024;
+  private static final double DATA_SIZE_SHIFT = 1024D * 1024;
```

Comment:
A little strange, why we use MB? (for User view, MB is ok, but in the program, why not use Byte?)